src/templates/account/login.html

```diff
@@ -9,6 +9,12 @@
 {% block heading %}{% trans "Sign In" %}<i class="pull-right fa fa-sign-in"></i>{% endblock %}
 
 {% block content %}
+  {% if request.GET.created %}
```

Comment:
Is there a reason we are adding the alert to the template instead of injecting it from the view like we do in the rest of the project?